src/main/java/mate/project/controller/BookController.java

```diff
@@ -35,7 +36,7 @@ public BookDto getBookById(@PathVariable Long id) {
     }
 
     @PostMapping
-    public BookDto createBook(@RequestBody CreateBookRequestDto bookDto) {
+    public BookDto createBook(@RequestBody @Valid CreateBookRequestDto bookDto) {
         return bookService.save(bookDto);
```

Comment:
In the method createBook, the return of bookService.save might return null, which would be a violation of the method's return type. Make sure to handle a possible null value.